waku/waku_relay/protocol.nim

```diff
@@ -130,6 +130,8 @@ type
     # a map that stores whether the ordered validator has been inserted
     # for a given PubsubTopic
     validatorInserted: Table[PubsubTopic, bool]
+    # seq of validators that are called for every pubsub topic
+    wakuDefaultValidators: seq[WakuValidatorHandler]
```

Comment:
not sure i understand this.
imho if rln is enabled, it should be enabled in all topics. same applies for all validators.

so not sure i understand the difference between validators and default validators. Having `addValidator` and `addDefaultValidator` creates some cognitive load for a use case that IMHO we don't have?